src/Github.php

```diff
@@ -177,6 +179,22 @@ public function setFullTime($value)
         return $this->updateQuery($fullTime, 'full_time');
     }
 
+    public function setCityStateLocation($job)
+    {
+        if (isset($job->location)) {
```

Comment:
This class should declare this `location` attribute. It is not inherited from parent class and will otherwise be public.